Update zha.markdown introduction for spelling and grammar + readability (#14689)
home-assistant/home-assistant.github.io: source/_integrations/zha.markdown

ha_domain: zha
---

> ZHA (Zigbee Home Automation) integration component for Home Assistant allows you to connect many off-the-shelf [Zigbee based devices](https://zigbeealliance.org) directly to Home Assistant, using one of the many available Zigbee coordinators that are compatible with [zigpy](https://github.com/zigpy/zigpy) (an open source Python library implementing a hardware independent Zigbee stack, which in turn relies on separate radio libraries to interface with Zigbee coordinator modules from different manufacturers).

Reviewer: - Removed the use of "component" we don't use that term in user-facing documentation.
- Removes an unneeded additional mention of "Home Assistant". This is the Home Assistant documentation, it is kinda implied.

```suggestion
The ZHA (Zigbee Home Automation) integration allows you to connect many off-the-shelf [Zigbee based devices](https://zigbeealliance.org) directly to Home Assistant, using one of the many available Zigbee coordinators.

ZHA uses an open-source Python library implementing a hardware-independent Zigbee stack called [zigpy](https://github.com/zigpy/zigpy). All coordinators compatible with zigpy can be used with ZHA.
```

I split the first line, mainly because it became long and wordy, and simplified the second part.

Author: Looks good to me!
